src/m365/spo/commands/group/group-member-remove.ts

```diff
@@ -80,19 +95,48 @@ class SpoGroupMemberRemoveCommand extends SpoCommand {
   }
 
   #initOptionSets(): void {
-    this.optionSets.push({ options: ['groupName', 'groupId'] });
+    this.optionSets.push(
+      { options: ['groupName', 'groupId'] },
+      { options: ['userName', 'email', 'userId'] }
+    );
+  }
+
+  private async getUserName(args: CommandArgs): Promise<string> {
+    if (args.options.userName) {
+      return args.options.userName;
+    }
+    else {
```

Comment:
You don't need this `else` statement here. You can remove it, it will make the code less layered.